cirq/sim/simulator.py

```diff
@@ -593,3 +605,13 @@ def _verify_unique_measurement_keys(circuit: circuits.Circuit):
         duplicates = [k for k, v in result.most_common() if v > 1]
         if duplicates:
             raise ValueError('Measurement key {} repeated'.format(",".join(duplicates)))
+
+
+def _check_all_resolved(circuit):
```

Comment:
This method isn't private anymore, so we can drop the leading underscore.

Reply:
done